assets/scss/utilities/_spacing.scss

```diff
@@ -56,4 +58,20 @@
 
 .has-reset-padding {
```

Comment:
Is this for elements that `normalize.css` fails to remove default user-agent padding from? If not, could we just remove this class?

Reply:
Way back [when](https://github.com/texastribune/queso-ui/pull/35), I needed some padding-less buttons. I thought maybe it'd be helpful to have this for instances where you just want to kill some padding on a component, but maybe we just leave that scoped to the component. I can create a padding-less button variation in that case.